CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 
 - Crash when OpenGL context resets
 - Modifiers being out of sync for fast/synthetic input on X11
+- Setting current working directory to non existent one when spawning a program or creating a window
```

Comment:
The thing that got fixed was that programs can't be spawned while inside a deleted directory, I don't think this is made entirely clear. I also think it should be possible to keep this a bit shorter, ideally under 80 columns, let me know if you need help with that.